packages/components/src/navigation/styles/navigation-styles.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import styled from '@emotion/styled';
+
+const horizontalPadding = '30px';
+
+export const Root = styled.div`
+	width: 272px;
+	padding: 35px 0px;
+	> *:not( .components-navigation__menu ) {
+		margin-left: ${ horizontalPadding };
+		margin-right: ${ horizontalPadding };
+	}
+`;
+
+export const Menu = styled.ul`
+	padding: 0;
+	margin: 0 0 32px 0;
+	display: flex;
+	flex-direction: column;
+`;
+
+export const MenuItem = styled.li`
```

Comment:
Would `MenuUI` be better here? When looking at the usage, it makes it seem like its actually a component, especially since there probably is a `MenuItem` in the components offering. I'm looking at this file for inspiration:
https://github.com/WordPress/gutenberg/blob/master/packages/components/src/card/styles/card-styles.js

Reply:
Nice!  I definitely like this pattern better; much more clear.  Updated in https://github.com/WordPress/gutenberg/pull/24522/commits/3cf1de0079d076c408aaa551fd4caf39bb3dc2e2